ajax/libs/monaco-editor/package.json

```diff
@@ -21,8 +21,7 @@
     {
       "basePath": "",
       "files": [
-        "min/**/*",
-        "min-maps/**/*"
+        "(min|min-maps)/**/*"
```

Comment:
Do you mind explaining to me what the difference between `(min|min-maps)/**/*` and `+(min|min-maps)/**/*` is?

Meanwhile, I'll ping @alexandrudima so we get to wrap this up.